actions/read_xcconfig_property.rb

```diff
@@ -8,12 +8,18 @@ class ReadXcconfigPropertyAction < Action
       # This is useful in case sensitive data stored in xcconfig files
       # is needed to be accessed from a fastlane script.
 
+      # Default xcconfig files path.
+      PROJECT_XCCONFIG_PATH = "%s/ConfigurationFiles/%s.xcconfig"
+
       def self.run(params)
-        if !File.exist?(params[:xcconfig_path])
+        xcconfig_path = PROJECT_XCCONFIG_PATH % [params[:project_name], params[:build_configuration]]
```

Comment:
But you are not sending `project_name` parameter in here: https://github.com/Wolox/fastlane-mobile/pull/3/files#diff-c41a4df45a632f3c6b44719dcfef8883L171

is that okay?

Reply:
EDIT: Just saw there is a default value in self.available_options